cli/task_runner.rs

```diff
@@ -204,7 +210,7 @@ impl ShellCommand for NpmCommand {
     mut context: ShellCommandContext,
   ) -> LocalBoxFuture<'static, ExecuteResult> {
     if context.args.first().map(|s| s.as_str()) == Some("run")
-      && context.args.len() > 2
+      && context.args.len() >= 2
```

Comment:
Why is it now `>=`?

Reply:
Turned out this code wasn't triggering. `context.args` doesn't include the command name (`npm`), so the first arg is `run` and the second arg is the name of the script. So `npm run foo` was missing this branch.